cg/models/flow_cell/flow_cell.py

```diff
@@ -45,15 +44,15 @@ class FlowCellDirectoryData:
     def __init__(self, flow_cell_path: Path, bcl_converter: str | None = None):
         LOG.debug(f"Instantiating FlowCellDirectoryData with path {flow_cell_path}")
         self.path: Path = flow_cell_path
-        self.machine_name: str = ""
+        self.machine_name: str = EMPTY_STRING
         self._run_parameters: RunParameters | None = None
         self.run_date: datetime.datetime = datetime.datetime.now()
         self.machine_number: int = 0
-        self.base_name: str = ""  # Base name is flow cell-id + flow cell position
-        self.id: str = ""
+        self.base_name: str = EMPTY_STRING  # Base name is flow cell-id + flow cell position
+        self.id: str = EMPTY_STRING
         self.position: Literal["A", "B"] = "A"
         self.parse_flow_cell_dir_name()
-        self.bcl_converter: str = self.get_bcl_converter(bcl_converter)
+        self.bcl_converter: str = bcl_converter or BclConverter.DRAGEN
```

Comment:
```suggestion
        self.bcl_converter: str = BclConverter.DRAGEN if not bcl_converter else bcl_converter 
```
not sure if just having or gets the wanted behaviour

Reply:
yes, this takes the value of the variable `bcl_converter` if it is not None, otherwise takes Dragen (BCLConvert)